drivers/bluetooth/hci/hci_tlx.c

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright (c) 2022, Telink Semiconductor (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/bluetooth/hci.h>
+
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/drivers/bluetooth/hci_driver.h>
+
+#include <b9x_bt.h>
```

Comment:
looks like you want to add new SoC, but keep old build like fake B92

Reply:
@andriy-bilynskyy @liucoldstar @damien0x0023 
In such case try to update/adopt new files as much as possible to simplify next porting
And we will take a look carefully only into changes file which can impact others SoCs
To avoid lots of PRs for each module/driver, maybe will be easily keep this PR and modules PRs till you will have more changes and after will merge all together into develop branches ?

Reply:
Thank you very much for your suggestions!
We will do more modificaiton and improvement based on your comments in andy's working branch tl321x_bringup.

This PR and my working branch tl321x_skeleton may be expired so we can ignore it.
Please DO NOT MERGE this PR because it's only very initial draft.